Water sensors code/Arduino code/MonitorReading.ino

```diff
@@ -0,0 +1,143 @@
+/*
+Liquid flow rate sensor -DIYhacking.com Arvind Sanjeev
+
+Measure the liquid/water flow rate using this code. 
+Connect Vcc and Gnd of sensor to arduino, and the 
+signal line to arduino digital pin 2.
+ 
+ */
+
+byte statusLed    = 13;
+byte check    = 4;
+
+
+byte sensorInterrupt = 0;  // 0 = digital pin 2
+byte sensorPin       = 2;
+
+// The hall-effect flow sensor outputs approximately 4.5 pulses per second per
+// litre/minute of flow.
+float calibrationFactor = 4.5;
+
+volatile byte pulseCount;  
+
+float flowRate;
+unsigned int flowMilliLitres;
+unsigned long totalMilliLitres;
+
+unsigned long oldTime;
+
+const byte numPins = 5;
+byte pins[] = { 12, 11, 10, 9,8};
+
+void setup()
+{
+  // Initialize a serial connection for reporting values to the host
+  Serial.begin(9600);
+   
+  // Set up the status LED line as an output
+  pinMode(statusLed, OUTPUT);
+   pinMode(check, OUTPUT);
```

Comment:
Your spacing is inconsistent throughout this file

Reply:
will chnage

Reply:
fixed this